mteb/models/qzhou_models.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+import os
+from functools import partial
+from mteb.model_meta import ModelMeta
+from mteb.encoder_interface import PromptType
+from mteb.models.instruct_wrapper import InstructSentenceTransformerWrapper
+
+def instruction_template(
+    instruction: str, prompt_type: PromptType | None = None
+) -> str:
+    if not instruction or prompt_type == PromptType.passage:
+        return ""
+    if isinstance(instruction, dict):
+        if prompt_type is None:
+            instruction = list(instruction.values())[0]  # TODO
+        else:
+            instruction = instruction[prompt_type]
+    return f"Instruct: {instruction}\nQuery:"
+
+
+
+def qzhou_instruct_loader(model_name, **kwargs):
+    model = InstructSentenceTransformerWrapper(
+        model_name,
+        revision=kwargs.pop("model_revision", None),
+        instruction_template=instruction_template,
+        apply_instruction_to_passages=False,
+        **kwargs,
+    )
+    encoder = model.model._first_module()
+    encoder.tokenizer.padding_side = "left"
+    return model
+
+
+QZhou_Embedding = ModelMeta(
+    loader = partial(
+        qzhou_instruct_loader,
+        model_name="Kingsoft-LLM/QZhou-Embedding",
+        model_revision="b43142d518d6e5251fd2d1e0a8741eef5c8b980a",
+    ),
+    name="Kingsoft-LLM/QZhou-Embedding",
+    languages=["eng-Latn", "zho-Hans"], 
+    open_weights=True,
+    revision="b43142d518d6e5251fd2d1e0a8741eef5c8b980a",
+    release_date="2025-08-01",
+    n_parameters=7_070_619_136,
+    memory_usage_mb=29070,
+    embed_dim=3584,
+    license="apache-2.0",
+    max_tokens=8192,
+    reference="https://huggingface.co/Kingsoft-LLM/QZhou-Embedding",
+    similarity_fn_name="cosine",
+    framework=["Sentence Transformers", "PyTorch"],
+    use_instructions=True,
+    public_training_code=None,
+    public_training_data="https://huggingface.co/cfli/datasets",
```

Comment:
As the user can add more datasets, this is not a valid reference. I would instead recommend that you document it in a section in the model card and then reference that section.